### PR TITLE
Pptp 1402 kgs

### PR DESCRIPTION
**Pull request by C-Thomp**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -166,24 +166,24 @@ convertedPackagingCredit.change.hidden = How much credit would you like to claim
...
+exportedPlasticPackagingWeight.heading = How much of your {0} of finished plastic packaging components did you export?
 exportedPlasticPackagingWeight.checkYourAnswersLabel = Weight of exported plastic packaging
 exportedPlasticPackagingWeight.error.nonNumeric = Weight must be entered as numbers only
 exportedPlasticPackagingWeight.error.required = Enter the weight, in kilograms
 exportedPlasticPackagingWeight.error.wholeNumber = Weight must not include decimals
 exportedPlasticPackagingWeight.error.spaces = Weight must not include spaces
 exportedPlasticPackagingWeight.error.outOfRange.low = Weight must be 0kg or more
 exportedPlasticPackagingWeight.error.outOfRange.high = Weight must be 99,999,999,999kg or less
-exportedPlasticPackagingWeight.change.hidden = How much of your {0} kg of finished plastic packaging components did you export?
+exportedPlasticPackagingWeight.change.hidden = How much of your {0}kg of finished plastic packaging components did you export?
```
Comment (liampan) [inline conf/messages.en:186]: shoud this kg be removed two, and the input value just call asKgs

File: conf/messages.cy [hunk 2]
```diff
@@ -148,7 +148,7 @@ amendRecycledPlasticPackaging.checkYourAnswersLabel = Weight of plastic packagin
 amendRecycledPlasticPackaging.error.nonNumeric = Amount must be entered as numbers only
 amendRecycledPlasticPackaging.error.required = Enter the weight, in kilograms
 amendRecycledPlasticPackaging.error.wholeNumber = Enter the weight using whole numbers
-amendRecycledPlasticPackaging.error.outOfRange = Amount cannot exceed 99,999,999 kgs
+amendRecycledPlasticPackaging.error.outOfRange = Amount cannot exceed 99,999,999kgs
```
Comment (liampan) [inline conf/messages.cy:151]: query if this is correct?

File: app/views/returns/DirectlyExportedComponentsView.scala.html [hunk 3]
```diff
@@ -34,7 +34,7 @@
     @paragraphBody(messages("directlyExportedComponents.paragraph.2"))
 }
 
-@layout(pageTitle = title(form, messages("directlyExportedComponents.title", totalPlastic), Some(messages("return.title.tail")))) {
+@layout(pageTitle = title(form, messages("directlyExportedComponents.title", totalPlastic.asKgs), Some(messages("return.title.tail")))) {
```
Comment (liampan) [inline app/views/returns/DirectlyExportedComponentsView.scala.html:37]: if kgs is incorrect coisider changing to asKg - however not a priority at all.

Comment (C-Thomp) [reply app/views/returns/DirectlyExportedComponentsView.scala.html]: done it anyway :)

File: conf/messages.cy (hunk 2 above)
Comment (C-Thomp) [reply conf/messages.cy]: Should be kg 👍

File: conf/messages.en (hunk 1 above)
Comment (C-Thomp) [reply conf/messages.en]: change.hidden messages updated with .asKg for the parameter

**Review by liampan — APPROVED** [review]

**Comment by C-Thomp** [comment]
AT's pass 👍

**Pull request merged**